### ISSUE TITLE
Rate of Change Indicator bug 

### ISSUE THREAD

**Issue by Jay-Jay-D** [QuantConnect/Lean#203]
[This line](https://github.com/QuantConnect/Lean/blob/master/Indicators/RateOfChange.cs#L57) should be:
```c#
return (input - window.MostRecentlyRemoved) / MostRecentlyRemoved;
```
If not is a simple difference / momentum instead of rate of change.

**Comment by mchandschuh** [comment]
That makes more sense to me as well. The source for our test data is [freestockcharts](www.freestockcharts.com). They have a Rate of Change indicator defined [here](http://www.freestockcharts.com/help/Content/Indicators/Rate_of_Change.htm).

After some searching, I couldn't find anyone else who defines Rate of Change like that.  [Here's](http://stockcharts.com/school/doku.php?id=chart_school:technical_indicators:rate_of_change_roc_and_momentum) one that suggests it should be a percentage, scaled 0 to 100, and then [this](http://www.investopedia.com/terms/p/pricerateofchange.asp) one suggests as you have defined.  We already have a `RateOfChangePercent` which defined [here](https://github.com/QuantConnect/Lean/blob/master/Indicators/RateOfChangePercent.cs#L59) and scaled 0 to 100, so let's update `RateOfChange` as you suggest.

The only question that remains: is there an external data source that has values we can test against?

**Comment by Jay-Jay-D** [comment]
IMHO, given the simplicity of the operation and the concept, an external data source to check the indicator is not needed.

As always, I'm glad to help.

**Comment by mchandschuh** [comment]
Closed via #250
